### PR TITLE
Disconnect arrow-incubator

### PR DESCRIPTION
**Pull request by danimontoya**
Disconnect arrow-optics-mtl and comment tests

### CODE REVIEW

File: arrow-optics/src/test/kotlin/arrow/optics/GetterTest.kt [hunk 1]
```diff
@@ -140,43 +129,43 @@ class GetterTest : UnitSpec() {
...
+//      forAll(genToken) { token ->
+//        tokenGetter.toState().run(token) == tokenGetter.extract().run(token)
+//      }
+//    }
+//
+//    "extractMap with f should be same as extract and map" {
+//      forAll(genToken, Gen.functionAToB<String, String>(Gen.string())) { token, f ->
+//        tokenGetter.extractMap(f).run(token) == tokenGetter.extract().map(f).run(token)
+//      }
+//    }
```
Comment (nomisRev) [inline arrow-optics/src/test/kotlin/arrow/optics/GetterTest.kt:169]: These can now be removed from here.

```suggestion
```

**Review by nomisRev — APPROVED** [review]
Looks great thanks @danimontoya! 🎉 
Looks like you forgot to remove tests in 1 file ;)

File: arrow-optics/src/test/kotlin/arrow/optics/GetterTest.kt (hunk 1 above)
Comment (danimontoya) [reply arrow-optics/src/test/kotlin/arrow/optics/GetterTest.kt]: yes!! thank you simon! Tests were moved.. but forgot to remove the comments! 👍

**Review by rachelcarmena — APPROVED** [review]
:tada:

**Pull request merged**